core/common/src/grpc_interceptor_samples.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+use bytes::Bytes;
+use core_protobuf_data_access::invehicle_digital_twin;
+use log::info;
+use prost::Message;
+use std::error::Error;
+
+use crate::grpc_interceptor::GrpcInterceptor;
+
+/// Sample gRPC interceptor.
+#[derive(Clone)]
+pub struct SampleGrpcInterceptor {}
+
+impl SampleGrpcInterceptor {
+    const INVEHICLE_DIGITAL_TWIN_SERVICE_NAME: &str = "InvehicleDigitalTwin";
+    const REGISTER_METHOD_NAME: &str = "Register";
+}
+
+#[allow(unused_variables)]
```

Comment:
Is this needed? Are you able to just prepend an underscore `_` to unused variables?

Reply:
Fixed.